### ISSUE TITLE
Staging Review finding: Unexpected keyboard focus behavior

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105695]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Education Data Migration
**Product:** WEAMS Public
**Feature:** WEAMS Public

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

There are a couple of places where keyboard focus behaves unexpectedly:
- On [/licenses-certifications-and-prep-courses](https://staging.va.gov/education/gi-bill-comparison-tool/licenses-certifications-and-prep-courses), when pressing "Reset search" focus stays on the reset button
- On [/licenses-certifications-and-prep-courses/results](https://staging.va.gov/education/gi-bill-comparison-tool/licenses-certifications-and-prep-courses/results), when pressing both "Update search" and "Reset search" focus stays on the pressed button

Proper management of keyboard focus servies two purposes:
- For sighted keyboard-only users, it ensures that the next tab stop is always a meaningful and useful tab stop --- meaning, after an action occurs you're placed somewhere that makes sense for whatever comes next in the user flow.
- For blind screen reader users, it determines what's announced immediately after an action takes place. If keyboard focus is lost or if it doesn't move to a next step that makes sense, screen readers often don't announce anything to their users. Silence can be disorienting and confusing, leaving the user to guess whether or not an action was successful.

## Recommended action

- On [/licenses-certifications-and-prep-courses](https://staging.va.gov/education/gi-bill-comparison-tool/licenses-certifications-and-prep-courses), send focus to the first input in the search form. (This is what's already happening on the [institution of higher learning programs page](https://staging.va.gov/education/gi-bill-comparison-tool/schools-and-employers/institution/31937132/institution-of-higher-learning).
- On [/licenses-certifications-and-prep-courses/results](https://staging.va.gov/education/gi-bill-comparison-tool/licenses-certifications-and-prep-courses/results), send focus to the "Showing X - Y..." message using `tabindex="-1"`.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jkdrahota** [comment]
<p>This issue has now been resolved and testing has passed. </p>
